### ISSUE TITLE
Kamalio version devel

### ISSUE THREAD

**Issue by faratell** [herlesupreeth/Kamailio_IMS_Config#39]
Hello. I want to create ims sever along with open5gs. i use https://open5gs.org/open5gs/docs/tutorial/02-VoLTE-setup/ but my os is ubuntu 22.04 and have been installed kamailio version devel from https://github.com/kamailio/kamailio. 
Can i use this link for  Copy pcscf, icscf and scscf configuration files to /etc folder for that?

**Comment by herlesupreeth** [comment]
yes, you can. You would have to use the files from master branch
